<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.exceptions.FieldExceptionSpec" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-05-04T14:13:43" hostname="testing-docker-ecc2246c-0b8e-4707-add3-97d74ea6771b" time="0.002">
  <properties/>
  <testcase name="Exception should have proper formatted message[0]" classname="griffon.exceptions.FieldExceptionSpec" time="0.002"/>
  <testcase name="Exception should have proper formatted message[1]" classname="griffon.exceptions.FieldExceptionSpec" time="0.0"/>
  <testcase name="Exception should have proper formatted message[2]" classname="griffon.exceptions.FieldExceptionSpec" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
